lib/models/constants.dart

```diff
@@ -1,3 +1,3 @@
 class Constants {
   static const url = 'https://shop-app-4ed38-default-rtdb.firebaseio.com/products.json';
```

Comment:
it would be better if you remove /products.json so that you could use this web address anywhere

this will be called your `endpoint`